jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/ResultFixtures.java

```diff
@@ -55,14 +55,28 @@ public static Result booleanFalseResult() {
 	}
 
 	public static Result numericResult() {
+		return numericResult(10);
+	}
+	public static Result numericResult(Object numericValue) {
 		return new Result(
 				0,
 				"ObjectPendingFinalizationCount",
 				"sun.management.MemoryImpl",
 				"ObjectDomainName",
-				"ObjectPendingFinalizationCount",
+				"MemoryAlias",
```

Comment:
In my test scenario I care about ensuring the alias was used.  I suppose I could have tested with the `ObjectPendingFinalizationCount` value, but given that this is the alias field it made sense to me to provide a distinct value from the attribute name.